scc/bls/bls_test.go

```diff
@@ -162,6 +162,18 @@ func TestPublicKey_String_IsHexEncoded(t *testing.T) {
 	require.Regexp(t, regexp, key.String())
 }
 
+func TestPublicKey_CanBeMarshalAndUnmarshal(t *testing.T) {
+	require := require.New(t)
+	key := PublicKey{}
+	data, err := key.MarshalJSON()
```

Comment:
This test should use `json.Marshal` and `json.Unmarshal`. Also the one for the signature below.